### DISCUSSION TITLE
p/2029b35ae4094a48a3073f998f10af9c/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#49] [Announcements]
# p/2029b35ae4094a48a3073f998f10af9c/

问题最近再次碰到 markdown 对中文的支持问题，由于这个问题已经长期存在，所以想谈一下现状和解决方法。当同时使用中文符号和粗体/斜体时，就会出现问题。 例如 **真没想到我这么快就要死了。**她有些自暴自弃地想着。 会被渲染为 **真没想到我这么快就要死了。**她有些自暴自弃地想着。 实际上，这种情况还有很多，几乎所有中文符号都无法被正常渲染。    示例 渲染    **真，**她 **真

https://blog.rxliuli.com/p/2029b35ae4094a48a3073f998f10af9c/

**Comment by Jack-Kingdom** [comment]
感觉更像是 markdown render 的 bug
markdown 的规范太简略了，每个实现都千奇百怪
